frontend-react/src/content/about/roadmap.mdx

```diff
@@ -19,21 +19,25 @@ import site from "../../content/site.json"
 
 <p className="usa-timestamp">Last updated: October 07, 2024</p>
```

Comment:
will the date be dynamically updated? If not, flagging to update to October 31, 2024 (assuming deployment happens on Thurs).